devops/auto_node/README.md

```diff
@@ -0,0 +1,35 @@
+# Docker image for sentry node deployment
+
+On any machine with docker (install [here](https://docs.docker.com/install/)) one could run a node, create a validator, and maintain it will 1 command.
+
+## Setup
+
+Curl `./scripts/auto_node.sh`, this is the main script that one can use to interact with the node.
+
+> For help on the parameters of `auto_node.sh` use the help option with `./auto_node.sh --help` 
+
+## Usages
+1. To run a node, use the `run` param of `auto_node.sh`. For example:
+```bash
+$ ./auto_node.sh run 6213bea7aef783463e67ed0c476a2915339de01f30658e7bb88ef5861e64b5e5  -N=stress -e=https://api.s0.stn.hmny.io/ -n=Harmony_Sentry_1 -s=1 -a -c -y
```

Comment:
what is key here?  Is it account key or blskey? please be a bit more explicit.

Reply:
Clarified in latest